verticapy/utilities.py

```diff
@@ -516,18 +529,29 @@ def pandas_to_vertica(
         if str_cols:
             tmp_df = df.copy()
             for c in str_cols:
-                tmp_df[c] = '"' + tmp_df[c] + '"'
+                tmp_df[c] = '"' + tmp_df[c].str.replace('\\','\\\\').str.replace('"','\\"') + '"'
             clear = True
         else:
             tmp_df = df
             clear = False
         import csv
 
         tmp_df.to_csv(path, index=False, quoting=csv.QUOTE_NONE, quotechar='', escapechar='\027',)
-        if (tmp_name):
-            vdf = read_csv(path, cursor, table_name=tmp_name, dtype=dtype, temporary_local_table=True, parse_n_lines=parse_n_lines,)
+        if (insert):
+            input_relation = "{}.{}".format(str_column(schema), str_column(name))
+            query = "COPY {}({}) FROM LOCAL '{}' DELIMITER ',' NULL '' ENCLOSED BY '\"' ESCAPE AS '\\' SKIP 1;".format(
+                input_relation,
+                ", ".join(['"' + col.replace('"', '""') + '"' for col in tmp_df.columns]),
+                path,
+            )
+            executeSQL(cursor, query, "Inserting the pandas.DataFrame.")
+            from verticapy import vDataFrame
+
+            vdf = vDataFrame(name, cursor=cursor, schema=schema)
+        elif (tmp_name):
+            vdf = read_csv(path, cursor, table_name=tmp_name, dtype=dtype, temporary_local_table=True, parse_n_lines=parse_n_lines, escape='\\',)
         else:
-            vdf = read_csv(path, cursor, table_name=tmp_name, dtype=dtype, schema=schema, temporary_local_table=False, parse_n_lines=parse_n_lines,)
+            vdf = read_csv(path, cursor, table_name=tmp_name, dtype=dtype, schema=schema, temporary_local_table=False, parse_n_lines=parse_n_lines, escape='\\',)
```

Comment:
`table_name=tmp_name`? Isn't `tmp_name == None`  here?

Reply:
Among my comments, I think only this one is critical. I approve the PR, but please resolve this before merging.

Reply:
Corrected, I will update the PR.